docs/notebooks/tutorials/EffectiveRipple.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>would prefer to show an example where the optimization actually improves things overall...</p><p> </p><p>also the 1.5 is formatted weird again</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
@f0uriest All of this will be replaced by #1290. Are the results I show in  https://github.com/PlasmaControl/DESC/blob/ku/fourier_bounce_neo/docs/notebooks/tutorials/EffectiveRipple.ipynb satisfactory? We can get better optimization results, but I wanted the tutorial to be runnable on laptop with cpu in a few minutes. Currently the optimization there has peak memory of 5gb.